lib/chalk-log.rb

```diff
@@ -104,6 +104,15 @@ def self.layout
     @layout ||= Chalk::Log::Layout.new
   end
 
+  # Adds a prefix to all logging within the current LSpace context.
```

Comment:
Ugh, I have so many feels about the interface to `chalk-log`. I'd much rather this involved instantiating a new logger in the desired context rather than passing things through LSpace but this is more consistent with how we write things today.